srcpkgs/stepmania/template

```diff
@@ -1,28 +1,35 @@
 # Template file for 'stepmania'
 pkgname=stepmania
-version=5.0.12
-revision=2
+version=5.1.0
+revision=1
+archs="i686 x86_64"
```

Comment:
Have you tried building for others? If it builds and their license doesn't forbid it, I don't see any issues. If it doesn't work, keep the comment about this being an upstream recommendation too.

Reply:
I believe their stance on this has changed, btw. At least there are open issues about ARM.